src/components/navigation-header/components/product-page-content/index.tsx

```diff
@@ -149,14 +150,12 @@ const ProductPageHeaderContent = () => {
 								{
 									label: 'Documentation',
 									iconColorTheme: currentProduct.slug,
-									items: currentProduct.rootDocsPaths.map(
-										(rootDocsPath: RootDocsPath) => {
-											return {
-												icon: rootDocsPath.iconName,
-												label: rootDocsPath.name,
-												path: `/${currentProduct.slug}/${rootDocsPath.path}`,
-											}
-										}
+									items: getDocsNavItems(currentProduct).map(
```

Comment:
Now using `getDocsNavItems` across `navigation-header` and `sidebar`'s `generate-product-landing-nav-items`.